packed_struct_codegen/Cargo.toml

```diff
@@ -7,13 +7,11 @@ license = "MIT OR Apache-2.0"
 authors = ["Rudi Benkovic <rudi.benkovic@gmail.com>"]
 
 [dependencies]
-quote = "^0.3.12"
+quote = "1.0"
```

Comment:
Consider using the caret here, as described in the [docs](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements)? Same holds true for syn and proc-macro2?

Reply:
The paragraph just above the linked docs says that if no operator is present then the caret is implied, so this should be effectively equivalent? It resolves to `quote v1.0.7` in the CI build right now, so it is working :)

Reply:
There we have it, now I should have done more research :)